src/bin/dump/softhsm2-dump-file.cpp

```diff
@@ -437,13 +437,15 @@ void dump(FILE* stream)
 			}
 			printf("(length %lu)\n", (unsigned long) len);
 
-			std::vector<uint8_t> value((size_t) len);
-			if (!readBytes(stream, value))
-			{
-				corrupt(stream);
-				return;
+			if (len > 0) {
```

Comment:
Nice. I just have some nits with the placement of the braces
```suggestion
			if (len > 0)
			{
```
which seems to be used in this file.

Reply:
Please do stylize as customary in the project, the wrong style came from my brain stem.